sources/libClangSharp/CMakeLists.txt

```diff
@@ -18,12 +18,6 @@ set(SOURCES
   CXType.h
 )
 
-set(LIBS
-  clangAST
-  clangFrontend
-  libclang
-)
```

Comment:
Can you provide more details about how you built such that you didn't have `clangAST` or `clangFrontend`?

These exist in both the official release binaries (https://github.com/llvm/llvm-project/releases/tag/llvmorg-12.0.0, see `lib/libclangAST.a and friends) and in the binaries produced if built locally.

Reply:
This CMake was also used to succesfully build libClangSharp for 12.0.0 on all supported platforms, so things were resolving correctly.

Reply:
> Can you provide more details about how you built such that you didn't have clangAST or clangFrontend?

sys-devel/clang version 12.0.0 from Gentoo main tree. Neither it nor the clang package from Arch provide those libraries. You can view the file list for the Arch package [here](https://archlinux.org/packages/extra/x86_64/clang/). 

The clang package here installs no static (.a) archives at all.

Reply:
[Gentoo configure](https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-devel/clang/clang-12.0.0-r1.ebuild#n237)
[Arch build script](https://github.com/archlinux/svntogit-packages/blob/packages/clang/trunk/PKGBUILD)

Reply:
This looks like its going to break windows as there is no `clang-cpp` produced.

I'm fine with taking it for Unix given it looks like the official binaries do create and ship `clang-cpp` by default.

Reply:
Updated adding library check.